CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 - [#1126](https://github.com/LayerManager/layman/issues/1126) Endpoint [GET Workspace Layer Thumbnail](https://github.com/LayerManager/layman/blob/v2.1.0/doc/rest.md#get-workspace-layer-thumbnail) was removed and replaced with endpoint [GET Layer Thumbnail](doc/rest.md#get-layer-thumbnail) endpoint to use UUID-based URL `/rest/layers/{uuid}/thumbnail` instead of workspace&name-based URL.
 - [#1126](https://github.com/LayerManager/layman/issues/1126) Endpoint [GET Workspace Map File](https://github.com/LayerManager/layman/blob/v2.1.0/doc/rest.md#get-workspace-map-file) was removed and replaced with endpoint [GET Map File](doc/rest.md#get-map-file) to use UUID-based URL `/rest/map/{uuid}/file` instead of workspace&name-based URL.
 - [#1126](https://github.com/LayerManager/layman/issues/1126) Endpoint [GET Workspace Layer Style](https://github.com/LayerManager/layman/blob/v2.1.0/doc/rest.md#get-workspace-layer-style) was removed and replaced with endpoint [GET Layer Style](doc/rest.md#get-layer-style) to use UUID-based URL `/rest/layer/{uuid}/style` instead of workspace&name-based URL.
+- [#1126](https://github.com/LayerManager/layman/issues/1126) Endpoint [Workspace Map](https://github.com/LayerManager/layman/blob/v2.1.0/doc/rest.md#workspace-map) (with methods GET, PATCH, DELETE) was removed and replaced with endpoint [Map](doc/rest.md#map) (with methods GET, PATCH, DELETE) to use UUID-based URL `/rest/maps/{uuid}` instead of workspace&name-based URL.
```

Comment:
In `metadata.md`, add second example of the `identifier` property around line 77. The second example should be identifier of the map with uuid-based url and also uuid in the label. Let's remove the first example when Workspace Layer endpoint is also migrated.

Also we forgot to change examples of `graphic_url` and `map_file_endpoint` properties to uuid-based URLs in one of previous PRs. Please update it in this PR.